edx_adapt/settings.py

```diff
@@ -1 +1 @@
-LOGS_DIR = '/var/log/edx-adapt/'
+LOGS_DIR = 'log/edx-adapt/'
```

Comment:
We should change permission for /var/log/edx-adapt dir in travis config or write logs into project dir while runing travis tests